### ISSUE TITLE
[Task]: Find and fix issue `failed to find seed vault`

### ISSUE THREAD

**Issue by nicole-obrien** [iotaledger/firefly#3693]
### Task description

Currently we have reports of this error and it completely corrupts the profiles. We should find and fix the cause of this. It seems to be happening only after first profile creation or restoration. If the user logs out and in before doing any further tasks it doesn't happen. But there is some steps that is causing this.

### Requirements

Find and fix the issue.

### Acceptance criteria

PR to fix is approved and merged into `stardust-develop`

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project
